keps/sig-node/2400-node-swap/README.md

```diff
@@ -561,6 +603,46 @@ type SwapStats struct {
 }
 ```
 
+In addition, we've added swap to stats to Summary API and Prometheus endpoints (`/stats/summary` and
```

Comment:
```suggestion
In addition, we'll add swap to stats to Summary API and Prometheus endpoints (`/stats/summary` and
```

This is framed as a proposal to enhance, and not a changelog. So the proposed changes should be future tense.